Gemfile.lock

```diff
@@ -156,7 +161,7 @@ GEM
     r2 (0.2.5)
     rack (1.6.0)
     rack-cors (0.3.1)
-    rack-openid (1.4.2)
+    rack-openid (1.3.1)
```

Comment:
Yes - see https://github.com/intridea/omniauth-openid/blob/master/omniauth-openid.gemspec which has "~> 1.3.1" so that only 1.3.x will be acceptable.